.swiftlint.yml

```diff
@@ -61,6 +119,43 @@ closure_end_indentation:
   severity: error
 closure_parameter_position:
   severity: error
+collection_alignment:
+  severity: error
+control_statement:
+  severity: error
+anyobject_protocol:
+  severity: error
+class_delegate_protocol:
+  severity: error
+cyclomatic_complexity:
+  severity: error
+syntactic_sugar :
+  severity: error
+file_header:
+  severity: error
+  required_pattern: |
+                  \/\*
+                   \* Copyright \(c\) \d{4}(-Present)?, Okta, Inc. and\/or its affiliates\. All rights reserved\.
+                   \* The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 \(the "License."\)
+                   \*
+                   \* You may obtain a copy of the License at http:\/\/www.apache.org\/licenses\/LICENSE-2.0.
+                   \* Unless required by applicable law or agreed to in writing, software
+                   \* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+                   \* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+                   \*
+                   \* See the License for the specific language governing permissions and limitations under the License.
+
+custom_rules:
+  multiple_empty_lines:
+      name: "Multiple Empty Lines"
+      regex: '((?:\s*\n){3,})'
+      message: "There are too many line breaks"
+      severity: error
+  empty_line_after_super:
```

Comment:
This might not work in cases where init is overridden, and `super.init` needs to be called at the end of the initializer.

Reply:
Right. Removed.